ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
+If you are submitting a bug issue:
+- [ ] Please include your platform type and number, as well as your Python, scikit-learn, numpy, and scipy versions.
```

Comment:
Not sure what "type and number" is. It would be good to provide a code snippet that prints all this out, perhaps:
```python
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import sklearn; print("Scikit-Learn", sklearn.__version__)
```